tool/update_proto.sh

```diff
@@ -14,7 +14,7 @@ BAZEL_REPO=.dart_tool/bazel_worker/bazel.git/
 rm -rf "$BAZEL_REPO"
 git clone --depth 1 https://github.com/bazelbuild/bazel.git "$BAZEL_REPO"
 
-protoc --proto_path="${BAZEL_REPO}/src/main/protobuf" --dart_out=lib/src worker_protocol.proto
+protoc --proto_path="${BAZEL_REPO}/src/main/protobuf" --dart_out="disable_constructor_args:lib/src" worker_protocol.proto
```

Comment:
Do we need this configuration option? It didn't seem to change anything?

Reply:
If it's standard now, I would actually be in favor of just regenerating with the new options and adding the constructors.  That might be worth justifying a release as well then.

Reply:
I'm all for adding the constructors! I regenerated the protos without the flag and updated existing code to use the constructors. Let me know how that looks :)